### ISSUE TITLE
Failure F26 links to the wrong SC?

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag#258]
From a reading of SC 1.3.3 I gather that it does not relate to images but to instructional text that
refer to other components identifying them by their shape or other
visual attribute.
In my opinion, F26: "Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information" does not seem to  have anything to do with 1.3.3. It is a failure of SC 1.1.1 because the symbol has no text alternative.

Thanks,
Sailesh

**Comment by patrickhlauke** [comment]
Coming in late, but - as this has recently come up in a discussion - I'd agree with Sailesh that F26 feels like a 1.1.1 failure.

**Comment by patrickhlauke** [comment]
bump... @alastc

**Comment by alastc** [comment]
It is closely related, but for 1.3.3 should really focus on the instuctions aspect, created #776 to update.

**Comment by electronicwoft** [comment]
does this mean that a new failure technique for conveying 'instructions for understanding and operating content' that rely on 'sensory characteristics' is required? That is, the intelligibility of icons, symbols, etc.?

**Comment by alastc** [comment]
Hi @electronicwoft, that's an option. It would be hard to specify (e.g. is the home symbol well enough known? Print? Where do you draw the line?).

There is another potential SC about [describing icons](https://github.com/w3c/wcag/issues/719#issuecomment-490442739), which might help with that.

**Comment by electronicwoft** [comment]
Hi @alastc ... what about a success criterion that uses a 'prescription of purposes' a la 1.3.5 for which there is a symbol from something like ISO graphical symbols? That is, if something performs A, then it is  represented by B. One problem I see with describing icons is the sense often given to 'describe' as  'exhaustively listing the characteristics of' such that what is signified is lost in  elucidation thereby being of little utility.
